### PR TITLE
feat: add new input component with additional 'set_value' method

### PR DESCRIPTION
**Pull request by sina-salahshour**

### CODE REVIEW

File: lua/zonkan/init.lua [hunk 1]
```diff
@@ -1,13 +1,52 @@
 local window = require("zonkan.window")
+local Input = require("zonkan.ui.input")
```
Comment (IdleSys) [inline lua/zonkan/init.lua:2]: In lua global variables start with capital letter. I think you might want to rename it.

File: lua/zonkan/init.lua [hunk 2]
```diff
@@ -1,13 +1,52 @@
...
+		default_value = "",
+		on_close = function()
+			print("Input closed!")
+		end,
+		on_submit = function(value)
+			print("Value submitted: ", value)
+		end,
+		on_change = function(value)
+			print("Value changed: ", value)
+			input:set_value("69")
```
Comment (IdleSys) [inline lua/zonkan/init.lua:39]: Never remove this line

**Review by IdleSys — COMMENTED** [review]
LGTM =)

File: lua/zonkan/init.lua (hunk 1 above)
Comment (sina-salahshour) [reply lua/zonkan/init.lua:2]: no i don't want to rename it 🚶🏻

Comment (IdleSys) [reply lua/zonkan/init.lua:2]: LGTM*2

**Pull request merged**